pages/patterns/delete.js

```diff
@@ -1,108 +1,126 @@
-import React from 'react';
+/* eslint-disable react/jsx-key */
+import React, { useState } from 'react';
 import { Heading } from '@twilio-paste/core/heading';
 import { Text } from '@twilio-paste/core/text';
 import { Paragraph } from '@twilio-paste/core/paragraph';
 import { useToaster, Toaster } from '@twilio-paste/core/toast';
+import { Alert } from '@twilio-paste/core/alert';
+import { Anchor } from '@twilio-paste/core/anchor';
+import { Box } from '@twilio-paste/core/box';
 
-import { DeleteTable } from '../../components/site/patterns/DeleteTable';
 import { DeleteConfirm } from '../../components/site/patterns/DeleteConfirm';
+import { DeleteTable } from '../../components/site/patterns/DeleteTable';
 
-export default function Delete() {
-  const severity = 'medium';
+/*
+ * To change severity level (for more information on that, check out Paste Delete Pattern docs),
+ * type 'low' or 'medium' below ('high' severity is not yet available).
+ */
```

Comment:
I like that this takes up a little more space now. Nice! I would maybe put the change the (for more information on that...) text to say (learn more about severity at paste.twilio.design/patterns/delete), since at first I thought the "that" you were referring to was the changing of the severity level.

Reply:
👍 ty!